pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/BrokerHostUsage.java

```diff
@@ -32,15 +41,26 @@
 public class BrokerHostUsage {
```

Comment:
What interface do we plan on having? Just exposing `SystemResourceUsage getBrokerHostUsage()` would be enough?